apps/sensenet/cypress/integration/ui/addNewMenu.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+context('AddNew Menu', () => {
+  before(() => cy.clearCookies({ domain: null } as any))
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('should open a dropdown with the list of allowed child types', () => {
+    const dropdownItems = [
+      'Folder',
+      'Document Library',
+      'Image Library',
+      'Event List',
+      'Memo List',
+      'Link List',
+      'Task List',
```

Comment:
```suggestion
      'Task list',
```
It is not your fault and it is only a minor thing but please change this text